data/data-pipeline/data_pipeline/etl/sources/tree_equity_score/etl.py

```diff
@@ -83,8 +83,52 @@ def transform(self) -> None:
             pd.concat(tes_state_dfs), crs=tes_state_dfs[0].crs
         )
 
+        # rename ID to Tract ID
+        self.df.rename(
+            columns={"geoid": ExtractTransformLoad.GEOID_FIELD_NAME},
+            inplace=True,
+        )
+
     def load(self) -> None:
         logger.info("Saving Tree Equity Score GeoJSON")
         # write nationwide csv
         self.CSV_PATH.mkdir(parents=True, exist_ok=True)
-        self.df.to_file(self.CSV_PATH / "tes_conus.geojson", driver="GeoJSON")
+        # here I remove the geometry
+        self.df = self.df[
+            [
+                ExtractTransformLoad.GEOID_FIELD_NAME,
```

Comment:
Because this whole ETL class is actually an extension of `ExtractTransformLoad`, you can just do `self.GEOID_FIELD_NAME`

Reply:
Good point - I will just reference the `super` constructor that is called first..